### PR TITLE
Update with the Polymer 1.1 style recommendations

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -22,16 +22,16 @@
...
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
+    "web-component-tester": "*",
+    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
     "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "*",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0"
+    "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
```
Comment (notwaldorf) [inline bower.json]: bower doesn't like trailing commas :(

**Comment by notwaldorf** [comment]
@abdonrd This looks great! Just a small comment on the `bower.json` file

File: bower.json (hunk 1 above)
Comment (abdonrd) [inline bower.json]: Ups! :sweat_smile: Updated!

**Comment by notwaldorf** [comment]
:gem:

**Pull request merged**